docs/content/utilities/typography.md

```diff
@@ -80,6 +80,8 @@ Change the line height density with these utilities. Responsive variants are als
 <p class="lh-condensed-ultra">
   Bacon ipsum dolor amet tri-tip chicken kielbasa, cow swine beef corned beef ground round prosciutto hamburger porchetta sausage alcatra tail. Jowl chuck biltong flank meatball, beef short ribs. Jowl tenderloin ground round, short loin tri-tip ribeye picanha filet mignon pig chicken kielbasa t-bone fatback. Beef ribs meatball chicken corned beef salami.
 </p>
+
+<style>.frame-example {height:350px;}</style>
```

Comment:
Maybe we should add a comment about this being a hack to fix a scrollbar issue so we don't forget later.

Reply:
@colebemis we could, but it would show up in the docs 😬

Reply:
a bunch of other docs pages are using this hack for various things, also something we should look into!

Reply:
Yeah, let's file an issue in the Doctocat repo to fix this if there isn't one already (I can do that!). 

Re: showing up in the docs – Wouldn't it be more confusing to have this in the docs with no context? I think it'd be helpful for people to know that this line is specifically for our docs website and is not required to get typography classes working in their project.

Reply:
Oh, just found the issue for this: https://github.com/primer/doctocat/issues/164#issuecomment-675602281

Should we just keep that issue open until we have a more permanent fix?

Reply:
yep!! and yeah i agree it's confusing, there are 6 other instances of folks using the `<style>.frame-example</style>` hack for different things, i'd like to get rid of all of them!